style.css

```diff
@@ -12,23 +12,86 @@ body {
   overflow-x: hidden;
 }
 
-.navbar {
-  background-color: white;
-}
+@media only screen and (max-width: 767px) {
+  header {
+    background-color: white;
+  }
 
-.navbar-content {
-  display: flex;
-  justify-content: space-between;
-  align-items: center;
-  margin-left: 40px;
-  margin-right: 40px;
-  margin-top: 15px;
-  border: none;
-}
+  .nav-link {
+    color: white;
+    text-decoration: none;
+    transition: 0.7s ease;
```

Comment:
- [optional] Great job fixing the width, spacing and color like the previous reviewer requested. please also add the following so it looks like the figma version.
   ```
 font-size: 32px;
    font-weight: bolder;
```